GetIntoTeachingApi/Controllers/TeachingEventBuildingsController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using GetIntoTeachingApi.Attributes;
+using GetIntoTeachingApi.Models;
+using GetIntoTeachingApi.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace GetIntoTeachingApi.Controllers
+{
+    [Route("api/teaching_event_buildings")]
+    [ApiController]
+    [Authorize(Roles = "Admin,GetIntoTeaching")]
+    public class TeachingEventBuildingsController : ControllerBase
+    {
+        private readonly IStore _store;
+
+        public TeachingEventBuildingsController(IStore store)
+        {
+            _store = store;
+        }
+
+        [HttpGet]
+        [CrmETag]
+        [PrivateShortTermResponseCache]
+        [Route("teaching_event_buildings")]
```

Comment:
This should be:

```
[Route("")]
```

Otherwise your path will be `/api/teaching_event_building/teaching_event_buildings`